pkg/chart/loader/load_test.go

```diff
@@ -255,22 +256,27 @@ sources:
 home: http://example.com
 icon: https://example.com/64x64.png
 `),
+			ModTime: time.Now(),
```

Comment:
Think we need a `c.ModTime` check in `TestLoadFiles`.

Reply:
Done!